src/audio/dai.c

```diff
@@ -561,10 +575,16 @@ static int dai_comp_trigger(struct comp_dev *dev, int cmd)
 	case COMP_TRIGGER_RELEASE:
 		/* only start the DAI if we are not XRUN handling */
 		if (dd->xrun == 0) {
-			/* recover the dma status */
-			ret = dma_release(dd->dma, dd->chan);
-			if (ret < 0)
-				return ret;
+			/* the dma pointer is syncd already at pause
+			 * during capture, donot need to set the
+			 * dma pointer again.
+			 */
+			if (dev->params.direction == SOF_IPC_STREAM_PLAYBACK) {
+				/* recover the dma status */
+				ret = dma_release(dd->dma, dd->chan);
```

Comment:
Where do we release() capture stream ?

Reply:
When the release command comes, we did not need to do the dma pointer again(by calling dma_release() function). because we adjust the dma pointer in dai_dma_cb() function already.